features/support/helpers.rb

```diff
@@ -106,21 +105,16 @@ def run_jekyll(args)
 
 # rubocop:disable Metrics/AbcSize
```

Comment:
can we remove the disabling of this check for this method now that it's been refactored?